openstack/glance/templates/etc/_glance-protection.pp.tpl

```diff
@@ -0,0 +1,12 @@
+[.*]
+create = pp:image_admin_only
+read = pp:image_viewer_only
+update = pp:image_admin_only
+delete = pp:image_admin_only
+
+
+[^upload_.*]
```

Comment:
Can you explain what do you want to achieve here? Afaik we are not using image properties that start with `upload_`, why protect them differently?

This may make sense for special properties like `hypervisor_type`, `hw_vif_model` etc.

Reply:
this section resolved https://ci.eu-de-2.cloud.sap/teams/core/pipelines/tempest/jobs/run-glance-tempest-plugin-in-qa-de-1/builds/72